chat/src/main/java/io/skygear/plugins/chat/ChatContainer.java

```diff
@@ -198,110 +210,93 @@ public void setConversationTitle(@NonNull final Conversation conversation,
      *
      * @param conversation the conversation
-     * @param adminId      the admin id
+     * @param adminIds      the admin ids
      * @param callback     the callback
      */
     public void removeConversationAdmin(@NonNull final Conversation conversation,
-                                        @NonNull final String adminId,
+                                        @NonNull final List<String> adminIds,
                                         @Nullable final SaveCallback<Conversation> callback) {
-        Set<String> adminIds = conversation.getAdminIds();
-        adminIds.remove(adminId);
-
-        this.setConversationAdminIds(conversation, adminIds, callback);
+        updateConversationMembership(conversation, "chat:remove_admins", adminIds, callback);
     }
 
-    /**
-     * Sets conversation participants.
-     *
-     * @param conversation   the conversation
-     * @param participantIds the participant ids
-     * @param callback       the callback
-     */
-    public void setConversationParticipants(@NonNull final Conversation conversation,
-                                            @NonNull final Set<String> participantIds,
-                                            @Nullable final SaveCallback<Conversation> callback) {
-        Map<String, Object> map = new HashMap<>();
-        String[] ids = new String[participantIds.size()];
-        participantIds.toArray(ids);
-        map.put(Conversation.PARTICIPANT_IDS_KEY, ids);
-
-        this.updateConversation(conversation, map, callback);
-    }
 
     /**
      * Add conversation participant.
      *
      * @param conversation  the conversation
-     * @param participantId the participant id
+     * @param participantIds the participant ids
      * @param callback      the callback
      */
-    public void addConversationParticipant(@NonNull final Conversation conversation,
-                                           @NonNull final String participantId,
+    public void addConversationParticipants(@NonNull final Conversation conversation,
+                                           @NonNull final List<String> participantIds,
                                            @Nullable final SaveCallback<Conversation> callback) {
-        Set<String> participantIds = conversation.getParticipantIds();
-        if (participantIds == null) {
-            participantIds = new HashSet<>();
-        }
-        participantIds.add(participantId);
-
-        this.setConversationParticipants(conversation, participantIds, callback);
+        updateConversationMembership(conversation, "chat:add_participants", participantIds, callback);
     }
 
     /**
      * Remove conversation participant.
      *
      * @param conversation  the conversation
-     * @param participantId the participant id
+     * @param participantIds the participant ids
      * @param callback      the callback
      */
     public void removeConversationParticipant(@NonNull final Conversation conversation,
```

Comment:
Please maintain `removeConversationParticipant` and add a `removeConversationParticipants`. One accept participantId, one accept participantIds. Same for Admin.